From: Peter Jones <pjones@redhat.com>
Subject: define abort to avoid an unnecessary reloc.

---
 Cryptlib/Include/OpenSslSupport.h |    1 +
 1 file changed, 1 insertion(+)

Index: b/Cryptlib/Include/OpenSslSupport.h
===================================================================
--- a/Cryptlib/Include/OpenSslSupport.h
+++ b/Cryptlib/Include/OpenSslSupport.h
@@ -380,5 +380,6 @@ extern FILE  *stdout;
 #define atoi(nptr)                        AsciiStrDecimalToUintn(nptr)
 #define gettimeofday(tvp,tz)              do { (tvp)->tv_sec = time(NULL); (tvp)->tv_usec = 0; } while (0)
 #define gmtime_r(timer,result)            (result = NULL)
+#define abort()
 
 #endif
